src/coffee/stockimport.coffee

```diff
@@ -383,8 +383,7 @@ class StockImport
                 debug "Error on handling 409 stock update error. Details: #{JSON.stringify(err)}"
                 Promise.reject err
           else
-            debug "Failed to retry the task after #{@max409Retries} attempts for stock #{JSON.stringify(entry)}"
-            Promise.reject err
+            Promise.reject new Error("Failed to retry the task after #{@max409Retries} attempts for stock #{JSON.stringify(entry)}")
```

Comment:
I dont think this should be part of this PR

Reply:
Also please do not swallow original error objects as they contain a lot of valuable informations. See my comment [here](https://github.com/sphereio/sphere-stock-import/pull/75#discussion_r193980177)

Reply:
We need here both the original error with its stack trace as also its context/custom message regarding amount of retries.